ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -562,26 +556,20 @@ function osd_activate {
     timeout 10  bash -c "while [ ! -e ${OSD_DEVICE} ]; do sleep 1; done"
     chown ceph. ${OSD_JOURNAL}
     if [[ ${OSD_DMCRYPT} -eq 1 ]]; then
-      ceph-disk -v --setuser ceph --setgroup disk --dmcrypt activate $(dev_part ${OSD_DEVICE} 1) ${CEPH_DISK_ACTIVATE_OPTS}
+      ceph-disk -v --setuser ceph --setgroup disk --dmcrypt activate --no-start-daemon $(dev_part ${OSD_DEVICE} 1)
```

Comment:
Jewel is not affected by this bug, this commit fixed it https://github.com/ceph/ceph-docker/pull/464/commits/d5ccca2bffaf6ac3a3b48374dfeed398129cb3f7